tests/unit/test-api.c

```diff
@@ -1002,6 +1017,28 @@ main (void)
     jerry_cleanup ();
   }
 
+  /* Test stopping an infinite loop. */
+  if (jerry_is_feature_enabled (JERRY_FEATURE_VM_EXEC_STOP))
+  {
+    jerry_init (JERRY_INIT_EMPTY);
+    const char *inf_loop_code_src_p = "while(true) {}";
+    parsed_code_val = jerry_parse ((jerry_char_t *) inf_loop_code_src_p, strlen (inf_loop_code_src_p), false);
+    TEST_ASSERT (!jerry_value_has_error_flag (parsed_code_val));
+
+    int countdown = 6;
+
+    jerry_set_vm_exec_stop_callback (vm_exec_stop_callback, &countdown, 16);
```

Comment:
a question to verify:
if the JS code do contains useful loop, e.g.
```
for (i = 0; i<6*16; i++)
{
 a[i] = process(i);
}
```
or multiple loops but the total VM_OC_BACKWARD_BRANCH bytecode exceeds 6*16, the case will also be stopped
here, right?